src/main/java/com/sigopt/model/Experiment.java

```diff
@@ -153,7 +153,7 @@ public Subresource(String prefix, String name, Class<T> klass) {
         }
 
         public APIMethodCaller<T> fetch(String id) {
-            return this.fetch().addParam("id", id);
+            return this.fetch().addPathComponent("id", id);
```

Comment:
https://sigopt.com/docs/endpoints/suggestions/detail and https://sigopt.com/docs/endpoints/observations/detail were actually broken before b/c of this line. Working now.